docs/sql-standards.md

```diff
@@ -32,6 +32,21 @@ DELETE FROM `creature_loot_template` WHERE `entry` = 3 AND `item` = 884;
 INSERT INTO `creature_loot_template` (`Entry`, `Item`, `Reference`, `Chance`, `QuestRequired`, `LootMode`, `GroupId`, `MinCount`, `MaxCount`, `Comment`) VALUES 
 (3, 884, 0, 40, 1, 1, 0, 1, 1, 'Comment');
 ```
+---
+
+When performing DELETE queries, always specify more than one field when possible.
```

Comment:
```suggestion
When performing DELETE queries, always specify more than one field when possible, in particular, specify the primary key present in the table.
```

Reply:
saying only "multiple fields" can be misunderstood or misleading

Reply:
Specifying another column/example as well as the primary key would likely be beneficial. In the case of the `creature` table, only the `guid` column is in the primary key, but the standard seems to be to specify another distinguishing column as well.

Reply:
How about this? Some tables have more than one primary key
```
When performing DELETE queries, always specify more than one field. In particular, specify at least one primary key present in the table and another field.
```

Reply:
about `creature` table, the concept is always the same, I think it's ok now